src/ode_solver/pod_galerkin_ode_solver.cpp

```diff
@@ -36,39 +120,105 @@ void PODGalerkinODESolver<dim,real,MeshType>::step_in_time (real dt, const bool
     //Galerkin projection, pod_basis = V
     //V^T*J*V*p = -V^T*R
 
-    pod->getPODBasis()->Tvmult(*this->reduced_rhs, this->dg->right_hand_side); // reduced_rhs = (pod_basis)^T * right_hand_side
-
-    pod->getPODBasis()->Tmmult(*this->reduced_lhs_tmp, this->dg->system_matrix); //reduced_lhs_tmp = pod_basis^T * system_matrix
-
-    this->reduced_lhs_tmp->mmult(*this->reduced_lhs, *pod->getPODBasis()); // reduced_lhs = reduced_lhs_tmp*pod_basis
-
-    solve_linear(
-            *this->reduced_lhs,
-            *this->reduced_rhs,
-            *this->reduced_solution_update,
-            this->ODESolverBase<dim,real,MeshType>::all_parameters->linear_solver_param);
-
-    pod->getPODBasis()->vmult(this->solution_update, *this->reduced_solution_update);
+    const Epetra_CrsMatrix epetra_system_matrix = this->dg->system_matrix.trilinos_matrix();
+    Epetra_Map system_matrix_rowmap = epetra_system_matrix.RowMap();
+    const Epetra_CrsMatrix epetra_pod_basis = pod->getPODBasis()->trilinos_matrix();
```

Comment:
Since `epetra_pod_basis` and `epetra_system_matrix` are bring created and deleted in every iteration, will it be more efficient to have them as members of the class (without const)?